configure.ac

```diff
@@ -358,14 +366,12 @@ VERSION=$PACEMAKER_VERSION
 # Detect highest API schema version (use git if available to list managed RNGs,
 # in case there are leftover schema files from an earlier build of a different
 # version, otherwise check all RNGs)
```

Comment:
This parenthesized comment belongs with the function now

Reply:
Yep